recipes/maggma/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "maggma" %}
+{% set version = "0.24.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/maggma-{{ version }}.tar.gz
+  sha256: c6349bd1419a974803aaa8abf5a5b0d0f5c94a1c61e86def881e79490e32c085
+  patches:
+    - msgpack.patch
+
+build:
+  number: 0
+  skip: true  # [win or py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - mgrant = mongogrant.scripts.mgrant:cli
```

Comment:
Wait, why is this delated in `maggma` and not `mongogrant`?

Reply:
Sorry - that is a mistake - it is fixed now.